datahub-web-react/src/app/entity/chart/getLogoFromPlatform.tsx

```diff
@@ -10,14 +7,5 @@ export function getLogoFromPlatform(platform: string) {
     if (platform === 'Looker') {
         return lookerLogo;
```

Comment:
looker remains?

Reply:
Dashboards and Charts do not have DataPlatformUrns. Thus there is no way we can push these from the server. My view is that we should migrate Charts + Dashboards to _also_ use DataPlatformUrns and then do the same here. Until then, this will unfortunately have to remain.